### PR TITLE
Add scrollbar to the notification menu

### PR DESCRIPTION
**Pull request by mouracamila**
This PR fixes the issues reported on #1147 enabling the scrollbar to the notifications menu.

Fixes: #1147

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by michalskrivanek** [comment]
ci add to whitelist

**Comment by michalskrivanek** [comment]
@mouracamila do you have a screenshot at hand, perhaps?

**Comment by mouracamila** [comment]
Screenshot from 2020-02-05 16-46-58

**Comment by michalskrivanek** [comment]
lgtm, @sjd78 @lwrigh good enough?

**Comment by lwrigh** [comment]
> lgtm, @sjd78 @lwrigh good enough?

It looks good to me!

**Comment by mouracamila** [comment]
I made some changes for the following purposes:

- To improve the Scroll in the notification list.
- To stationary the "Clear all" button at the bottom of the message list.

I tried to use the model that I found here: https://patternfly-react-pf3.surge.sh/?path=/story/patternfly-react-communication-notification-drawer--notification-drawer

But, after some tests and read the documentation about NotificationDrawer and PanelBody (etc..) it seem that the format of the NotificationDrawer doesn't allow me to do the required changes using only the Patternfly's components. Therefore, I found no other solution beyond added the mensage list inside a div.

**Comment by mouracamila** [comment]
Screenshot from 2020-02-11 21-48-38

**Comment by sjd78** [comment]
ci test

File: src/components/VmUserMessages/style.css [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.drawer-pf-notifications-list {
+    border: solid 1px #d1d1d1 !important;
+    padding: 10px 0 !important;
+    margin: 0;
+    overflow-y: auto;
+}
```
Comment (sjd78) [inline src/components/VmUserMessages/style.css]:
```suggestion
.notifications-list {
    overflow-y: auto;
}
```

With the way css is import to the components, we can use shorter class names.

The border, padding and margins aren't necessary.  The extra padding makes the empty state look strange.

File: src/components/VmUserMessages/style.css [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+
+.drawer-pf-action-clear-notification-list {
+    flex: auto;
+    overflow-y: auto;
```
Comment (sjd78) [inline src/components/VmUserMessages/style.css]:
```suggestion
.action-panel {
    border-top: solid 1px #d1d1d1;
```

Only the extra border style is needed if the class itself is attached to the parent component it is currently attached to.

File: src/components/VmUserMessages/index.js [hunk 3]
```diff
@@ -85,10 +85,12 @@ class VmUserMessages extends React.Component {
         <NotificationDrawer hide={!this.state.show} expanded={this.state.expanded}>
           <NotificationDrawer.Title onCloseClick={this.handleToggle} onExpandClick={this.handleExpand} />
           <NotificationDrawer.PanelBody className={style['panel-body']}>
-            {messagesList}
+            <div className={style['drawer-pf-notifications-list']}>
```
Comment (sjd78) [inline src/components/VmUserMessages/index.js]:
```suggestion
            <div className={style['notifications-list']}>
```
Shorter class names with this kind of css import.

File: src/components/VmUserMessages/index.js [hunk 4]
```diff
@@ -85,10 +85,12 @@ class VmUserMessages extends React.Component {
...
+              {messagesList}
+            </div>
             { messagesCount > 0 &&
               <NotificationDrawer.PanelAction>
-                <NotificationDrawer.PanelActionLink data-toggle='clear-all'>
+                <NotificationDrawer.PanelActionLink data-toggle='clear-all' className={style['drawer-pf-action-clear-notification-list']}>
```
Comment (sjd78) [inline src/components/VmUserMessages/index.js]: Move the `className` up to the `PanelAction` instead of here on `PanelActionLink`.

**Pull request merged**